OnlineShopUI/online-shop-ui/src/components/ModalLogin/Login.js

```diff
@@ -0,0 +1,94 @@
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import CloseIcon from '@mui/icons-material/Close';
+import './Login.scss';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import User from '../../img/icon-user.png';
+import { Link } from 'react-router-dom';
+import useLogin from './Logic/LoginLogic';
+import Handlers from '../../Helper/Handlers';
+
+export default function Login() {
+  const [setUsername, setPassword, flag, setFlag] = useLogin();
+
+  const [handleLogInModalOpen, handleLogInModalClose, isOpen] = Handlers();
+
```

Comment:
Rename custom hook to start with 'use'. Modal window doesn't open with such a hook